flytekit/clis/sdk_in_container/run.py

```diff
@@ -399,7 +399,10 @@ def convert_to_struct(
         Convert the loaded json object to a Flyte Literal struct type.
         """
         if type(value) != self._python_type:
-            o = cast(DataClassJsonMixin, self._python_type).from_json(json.dumps(value))
+            if hasattr(self._python_type, "parse_raw"):  # e.g pydantic basemodel
```

Comment:
Question: This is for passing base models as json string to `pyflyte run`? (Just to make sure I understand.)

Reply:
Correct!